app-doc/gimp-help/gimp-help-2.10.0.ebuild

```diff
@@ -0,0 +1,33 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python2_7 )
+inherit python-any-r1
+
+DESCRIPTION="GNU Image Manipulation Program help files"
+HOMEPAGE="https://docs.gimp.org/"
+SRC_URI="mirror://gimp/help/${P}.tar.bz2"
+
+LICENSE="FDL-1.2"
+SLOT="2"
+KEYWORDS="~alpha ~amd64 ~hppa ~ia64 ~ppc ~ppc64 ~sparc ~x86"
+IUSE=""
+
+BDEPEND="${PYTHON_DEPS}
+	sys-devel/gettext
+"
+
+DEPEND="$(python_gen_any_dep 'dev-libs/libxml2[python,${PYTHON_USEDEP}]')
+	dev-libs/libxslt
+"
+
+src_configure() {
+	econf --without-gimp
+}
+
+src_compile() {
+	# see https://bugs.gentoo.org/677198
+	emake -j1
```

Comment:
I took this off and it build succesfully, however without digging into upstream makefiles I believe it's limited there. Because it was rather slow... but I'll take it off on a separate commit, taking the blame. If you get bug reports, just put it back :P